### ISSUE TITLE
bug: Clarification on merged SNIP status

### ISSUE THREAD

**Issue by boyuanx** [starknet-io/SNIPs#88]
Currently, most if not all merged SNIPs still have their statuses as "Draft". I'm wondering if this is intentional, or should they be "Final"?

**Comment by dor-starkware** [comment]
Hi @boyuanx,

I plan to review all the SNIPs in the main branch and ensure that the authors update their statuses to the appropriate ones according to SNIP-1 guidelines. Thank you for bringing this to my attention. I'll make sure we address this as soon as possible.

**Comment by boyuanx** [comment]
@dor-starkware I also noticed a lot of SNIP numbers do not properly adhere to the corresponding PR number, as specified in SNIP-1

**Comment by dor-starkware** [comment]
Hi @boyuanx,

Thank you for your observations. Just to clarify, are you noting a discrepancy between the SNIP numbers and the corresponding PR numbers in the main branch?

According to SNIP-1 guidelines:

> Once the SNIP is ready for the repository, the SNIP editor will:
> Assign a SNIP number (generally the PR number, but the decision is with the editors)

This indicates that while the SNIP number is generally the same as the PR number, the final decision lies with the editors, and they may assign a different number if deemed necessary. When I merge a new PR, I assign it a SNIP number in a sequential manner to make the SNIPs more organized.
